### PR TITLE
Template files for grading and project work

### PR DESCRIPTION
**Pull request by rhazn**
Added:

- automated grading feedback ci file and explanation
- example data
- example data exploration notebook
- example report notebook
- updated readme to include explanations of the added files

Not sure if we want to go this "deep" with example files or leave the notebooks out? Also we could include a testing file and automated ci action file that is a no-op before students use it... but not sure that is good or confusing. Let me know what you think.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# AMSE/SAKI 2023 Template Project
+This template project provides some structure for your open data project in the AMSE/SAKI module.
+
```
Comment (georg-schwarz) [inline README:3]:
```suggestion
This repository contains (a) a data science project that is developed by the student over the course of the semester, and (b) the exercises that are submitted over the course of the semester.
```

File: README [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+## Project Setup
+The following files are part of this template repository as examples and should be **replaced by you** over the semester:
+
+- `data.sqlite`: Your final, cleaned dataset. You will create an automated data pipeline that creates this SQLite database from multiple open data sources. The template repository includes data about train stations in germany, you need to replace this with your data!
+- `exploration.ipynb`: A Jupyter notebook that you can use to explore your data and show in detail what it looks like. You can refer to this file in your report for users that want more information about your data.
+- `report.ipynb`: Your final report as a Jupyter notebook. This is the result of your project work and should lead with a question that you want to answer using open data. The content of the report should answer the question, ideally using fitting visualizations, based on the data in `data.sqlite`.
```
Comment (georg-schwarz) [inline README:11]: Love it that we have separate notebooks!

File: README [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+
+## Exercises
+During the semester you will need to complete exercises, sometimes using [Python](https://www.python.org/), sometimes using [Jayvee](https://github.com/jvalue/jayvee). You **must** place your submission in the `exercises` folder in your repository and name them according to their number from one to five: `exercise<number from 1-5>.<jv or py>`. At the end of the semester, you must have the following files in your repository for full marks:
```
Comment (georg-schwarz) [inline README:15]: Last sentence potentially leaves a wrong impression that there is a final submission at the end of the semester. Also would note that they will be informed separately if they should do jv or python in the corresponding week.

File: exploration.ipynb [hunk 4]
```diff
@@ -0,0 +1,365 @@
+{
```
Comment (georg-schwarz) [inline exploration.ipynb:1]: Didn't review the notebook, but I assume it is okay.

**Review by georg-schwarz — APPROVED** [review]
LGTM. Left some input for wording improvements.

File: README [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+1. `./exercises/exercise1.jv` or `./exercises/exercise1.py`
+2. `./exercises/exercise2.jv` or `./exercises/exercise2.py`
+3. `./exercises/exercise3.jv` or `./exercises/exercise3.py`
+4. `./exercises/exercise4.jv` or `./exercises/exercise4.py`
+5. `./exercises/exercise5.jv` or `./exercises/exercise5.py`
+
+### Grading Feedback
+We provide automated grading feedback using a GitHub action (that is defined in `.github/workflows/grading-feedback.yml`). To view your grading feedback, navigate to Actions -> Grading Feedback in your repositort (or use the direct link [/actions/workflows/grading-feedback.yml](/actions/workflows/grading-feedback.yml)).
```
Comment (felix-oq) [inline README:24]: Typo:
```suggestion
We provide automated grading feedback using a GitHub action (that is defined in `.github/workflows/grading-feedback.yml`). To view your grading feedback, navigate to Actions -> Grading Feedback in your repository (or use the direct link [/actions/workflows/grading-feedback.yml](/actions/workflows/grading-feedback.yml)).
```

File: README [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+The grading feedback is executed whenever you make a change in files in the `exercise` folder. To see the feedback, open the lates GitHub Action run, open the `grading-feedback` job and `Grading feedback` step. You should see command line output that contains output like this:
```
Comment (felix-oq) [inline README:26]:
```suggestion
The grading feedback is executed whenever you make a change in files in the `exercise` folder. To see the feedback, open the latest GitHub Action run, open the `grading-feedback` job and `Grading Feedback` step. You should see command line output that contains output like this:
```

**Review by felix-oq — APPROVED** [review]
I agree on Georgs points. Otherwise I only found a few typos.

I like that the repository is filled with an example, so it is easy to understand which files belong where and what their purposes are.

**Pull request merged**